convert_csv_records_to_database.py

```diff
@@ -85,5 +100,5 @@ def main():
     convert_csv_file_to_db(args.csv_file_path, args.output_db_path)
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
     main()
```

Comment:
wrap with `sys.exit(...)`

Reply:
`main` doesn't return anything and I'm too lazy to catch all the possible exceptions and return errors for them

Reply:
That's... how `sys.exit` works.. not returning an error means success.